.github/workflows/ci.yaml

```diff
@@ -452,10 +460,18 @@ jobs:
           key: ${{ runner.os }}-general-${{ hashFiles('**/build.sbt') }}
           restore-keys: ${{ runner.os }}-general-
 
-      - name: Nightly Documentation
+      - name: Generate Website
```

Comment:
Do we need to do this on releases instead of just doing it on nightly? Doing it on releases is problematic because we release old versions too: https://github.com/lampepfl/dotty/issues/8831

Reply:
#8831 is not a problem since it's a human error rather than a software bug. Once CI is able to publish at all, the possibility for such errors will go away.

Reply:
We never had such problems on drone which was able to publish

Reply:
Ah, so the website is not updated when the release is created?

Reply:
No, I updated it manually. If you look at the ci logs e.g. from a nightly, you'll see that the website publish task gives a false positive. This PR is supposed to resolve this.